packages/react-core/src/components/NotificationBadge/examples/NotificationBadgeWithCount.tsx

```diff
@@ -2,34 +2,44 @@ import React from 'react';
 import { NotificationBadge, NotificationBadgeVariant } from '@patternfly/react-core';
 
 export const NotificationBadgeWithCount: React.FunctionComponent = () => {
-  const [firstVariant, setFirstVariant] = React.useState(NotificationBadgeVariant.unread);
-  const [firstVariantCount, setFirstVariantCount] = React.useState(30);
-  const [secondVariant, setSecondVariant] = React.useState(NotificationBadgeVariant.attention);
-  const [secondVariantCount, setSecondVariantCount] = React.useState(30);
+  const [readExpanded, setReadExpanded] = React.useState(false);
+  const [unreadExpanded, setUnreadExpanded] = React.useState(false);
+  const [attentionExpanded, setAttentionExpanded] = React.useState(false);
 
-  const onFirstClick = (value: NotificationBadgeVariant) => {
-    setFirstVariant(value);
-    setFirstVariantCount(10);
+  const onReadClick = () => {
+    setReadExpanded(!readExpanded);
   };
 
-  const onSecondClick = (value: NotificationBadgeVariant) => {
-    setSecondVariant(value);
-    setSecondVariantCount(10);
+  const onUnreadClick = () => {
+    setUnreadExpanded(!unreadExpanded);
+  };
+
+  const onAttentionClick = () => {
+    setAttentionExpanded(!attentionExpanded);
   };
 
   return (
     <div className="pf-t-dark">
       <NotificationBadge
-        variant={firstVariant}
-        onClick={() => onFirstClick(NotificationBadgeVariant.read)}
-        aria-label="First notifications"
-        count={firstVariantCount}
+        variant={NotificationBadgeVariant.read}
+        onClick={onReadClick}
+        aria-label="Notification badge with count and read variant"
```

Comment:
Just a nit but I might name these more consistently between the two examples. Something like

Basic - `aria-label="Notification badge with [read/unread/attention] variant"`
Count - `aria-label="Notification badge with count and [read/unread/attention] variant"`

or

Basic - `aria-label="[Read/Unread/Attention] notification badge"`
Count - `aria-label="[Read/Unread/Attention] notification badge with count"`